src/scss/skin-modern/_variables.scss

```diff
@@ -20,3 +20,5 @@ $subtitle-text-border-color: #000 !default;
 
 $animation-duration: .3s !default;
 $animation-duration-short: $animation-duration / 2 !default;
+
+$seekbar-height: 5px;
```

Comment:
Working with pixel values is a problem because the volume and seekbars don't scale with the UI, e.g. when setting `font-size: 2em` on the container to scale the UI to 200%.

Reply:
True but when we would change this to `em`, i guess then we would have the old issue again because we wont get the pixel-perfect design for the seek-bar. 

For example: 
Currently the size of the position marker is `$seekbar-height * 3 - 4px`. It's `4px` because it's preferred to be an even number (why? described in the PR description). 

`4px` -> `.25em` with `16px` default size. When changing the size to `$seekbar-height * 3 - .25em` everything looks fine.

But when changing the default size to `20px` then `.25em` will result in -> `5px`. And it looks weird again.

Reply:
So are you saying that this purely CSS-based approach will not yield any advantages over the current SVG-based approach if we don't use absolute pixel sizes? (And in turn we could use pixel sizes with the SVG approach to achieve the same result?)

Reply:
True. What if we calculate the `$seekbar-height` based on `em` and convert it to `px`?

```scss
@function em2px($target, $context: $font-size) {
  @if $target == 0 { @return 0 }
  @return ($target / 1em) * ($context / 1em) * 16 + 0px;
}

$seekbar-height: em2px(.3125em);
```

This would support the UI scale with changing the `$font-size`.

Reply:
I don't think it's possible to know the font-size at compile time. This would have to be implemented as a native CSS calculation?

Reply:
That's also true this will only support our `$font-size` variable.